pallets/utxo/src/tests.rs

```diff
@@ -516,3 +516,53 @@ fn test_send_to_address() {
         ));
     })
 }
+
+#[test]
```

Comment:
I feel like the tests are not enough at all... there has to be tests also for all the negative cases, not only positive cases. Also tests that using fuzzing to prove that a data model is correct are encouraged.

Reply:
As far as I remember, Ethereum uses for NFT only 3 tests. We have a bunch of tests for simple transactions. If we make a wrong transaction it will break other tests. Also, Substrate won't allow you to send the wrong types to your `pallet::call`. I will think about what we can test more but all things come to these pieces of code that have already been covered.

Reply:
If I understand correctly, we're writing the token protocol from scratch. There has to be tests for higher and lower amount and equal amount of what exists. Tests for combining inputs and splitting outputs. There has to be tests for wrong token IDs. For too big data, too small data, invalid formats, etc. There's lots to be tested. This test is not enough, not by a long shot.